cms/publications/tests.py

```diff
@@ -4,6 +4,47 @@
 from cms.publications.models import Publication, PublicationIndexPage, TOC
 
 
+class TestDocumentContainer(TestCase):
+    fixtures = ["fixtures/document_container.json"]
+
+    def setUp(self):
+        response = self.client.get("/pub-index-page/publication-with-attachments/")
+        soup = BeautifulSoup(response.content, "html.parser")
+        self.external_url, self.internal_page, self.pdf, self.docx, self.unknown = [
+            str(attachment) for attachment in soup.select(".attachment")
+        ]
+
+    def test_has_bytes(self):
```

Comment:
It's not immediately clear what this is actually testing for, so it might be worth a clarifying comment.